### ISSUE TITLE
Reiter Abfülldaten: Reihenfolge Freitextfelder

### ISSUE THREAD

**Issue by Rasinari** [kleiner-brauhelfer/kleiner-brauhelfer-2#270]
Ich stolpere soeben über die Freitextfelder im Reiter Abfülldaten.

Meines Erachtens müssten es 3 Felder sein, in dieser Reihenfolge - gelesen von unten nach oben:

1. Gärung
2. Abfüllen
3. Reifung / Nachgärung

Ich habe hier ein anspruchsvolles DIPA mit 2. Kaltstopfungen, wobei die 2. Kalthopfung auf einen 2. Fermenter erfolgen muss, bevor zum Cold Crash auf ein 3. Keg umgefüllt wird. Dieses Vorgehen erfordert eine saubere Trennung zwischen Gärung (1. Hopfenstopfen) und 2. und 3, wobei ich hier das 2. Hopfenstopfen dem Abfüllen zuschlagen würde.

**Comment by BourgeoisLab** [comment]
Ich finde ein drittes Freifeld für die Reifung etwas overkill. Das wird wahrscheinlich ganz selten benutzt. Ich könnte die Bezeichnung von "Bemerkung Gräung" nach "Bemerkung Gärung/Reifung" ändern. Im Text kann man ja immer noch selber unterscheiden.

**Comment by Rasinari** [comment]
Ja, das stimmt. Die beiden zusammenfassen macht Sinn. Abfüllen würde ich dann aber nach unten setzen - oder man findet zwei Begriffe für Abfüllen: wie wäre es mit Abfüllen und Umfüllen? Oder ist Umfüllen ein Teil der Reifung? Bin mir da nicht sicher…
